Source/Core/Core/Core.cpp

```diff
@@ -535,15 +516,20 @@ static void EmuThread(std::unique_ptr<BootParameters> boot)
     Keyboard::LoadConfig();
   }
 
+  const std::optional<std::string> savestate_path = boot->savestate_path;
```

Comment:
A const ref would be fine too, no?

Reply:
No, because we `move` `boot` further down in the function.